pkg/driver/controller.go

```diff
@@ -951,7 +952,7 @@ func newCreateVolumeResponse(disk *cloud.Disk, ctx map[string]string) *csi.Creat
 	}
 }
 
-func newCreateSnapshotResponse(snapshot *cloud.Snapshot) (*csi.CreateSnapshotResponse, error) {
```

Comment:
Why are moving the error out of this function and into the return statement of other functions? Does some linter complain?

Reply:
make verify was throwing an error as the error from the return is always nil